!Job
target: !Target
  head: dbd380d39dcf16207d3644d75e403e8359c4d731
  email: unknown
  branch: ARROW-17108
  remote: https://github.com/raulcd/arrow.git
  github_repo: raulcd/arrow
  version: 9.0.0.dev408
  no_rc_version: 9.0.0.dev408
  no_rc_semver_version: 9.0.0-dev408
tasks:
  test-conda-python-3.9-dask-latest: !Task
    ci: github
    template: docker-tests/github.linux.yml
    artifacts: []
    params:
      env:
        PYTHON: 3.9
        DASK: latest
      # use the latest dask release, so prevent reusing any cached layers
      flags: --no-leaf-cache
      image: conda-python-dask
    branch: actions-b0bee80d63-github-test-conda-python-3.9-dask-latest
    commit: 744207f49804827977bf7118259e066d408f121b
  test-conda-python-3.9-dask-master: !Task
    ci: github
    template: docker-tests/github.linux.yml
    artifacts: []
    params:
      env:
        PYTHON: 3.9
        DASK: master
      # use the latest dask release, so prevent reusing any cached layers
      flags: --no-leaf-cache
      image: conda-python-dask

# Turbodbc is currently failing to build. See:
# https://issues.apache.org/jira/browse/ARROW-15997
# #   test-conda-python-3.7-turbodbc-latest:
#     ci: github
#     template: docker-tests/github.linux.yml
#     params:
#       env:
#         PYTHON: 3.7
#         TURBODBC: latest
#       # use the latest turbodbc release, so prevent reusing any cached layers
#       flags: --no-leaf-cache
#       image: conda-python-turbodbc
# #   test-conda-python-3.7-turbodbc-master:
#     ci: github
#     template: docker-tests/github.linux.yml
#     params:
#       env:
#         PYTHON: 3.7
#         TURBODBC: master
#       # use the latest turbodbc release, so prevent reusing any cached layers
#       flags: --no-leaf-cache
#       image: conda-python-turbodbc
# 
    branch: actions-b0bee80d63-github-test-conda-python-3.9-dask-master
    commit: bdf43c1f714b5bfcb7a0aa4cb2b748d8f86db60d
params: {}
branch: actions-b0bee80d63
